src/components/Main.js

```diff
@@ -195,8 +195,8 @@ export default function Main() {
               Welcome to Sommelier the <span >coprocessor</span><br />for Ethereum
```

Comment:
@rootedbox this was supposed to say **"new coprocessor"** as it was in the spreadsheet.